azure-stack/hci/deploy/deployment-tool-install-os.md

```diff
@@ -28,14 +28,17 @@ Before you begin, make sure you've done the following:
 
 ## Boot and install the operating system
 
-To install the Azure Stack HCI operating system, follow these steps:
+The Supplemental package supports only the English version of the Azure Stack HCI operating system.
+
```

Comment:
Perhaps reword: Azure Stack HCI Supplemental Package supports only the English version of the Azure Stack HCI operating system.

Reply:
@ManikaDhiman @alkohli - if you want to make these changes, can you submit them in a new pr? thanks.